image/cli/mascli/functions/pipeline_config_advanced

```diff
@@ -48,7 +48,10 @@ function config_pipeline_advanced() {
       prompt_for_input "+ MongoDB Namespace" MONGODB_NAMESPACE "mongoce"
     fi
     echo
-
+    if [[ -z "$OCP_INGRESS_TLS_SECRET_NAME" ]]; then
+      OCP_INGRESS_TLS_SECRET_NAME="router-certs-default"
```

Comment:
I think the default here should still be "" ... most users - even when doing the advanced settings don't want to have to set this secret.

Probably have a: `Override the ingress certificate lookup? y/N` prompt .. so that most people can just click though and retain the same logic we have today.

Reply:
Ok, I'll change the logic to ask if user really wants to override this prior prompting them to type the cert secret.

Reply:
<img width="1617" alt="image" src="https://github.com/ibm-mas/cli/assets/31037381/3f387b50-6e19-4edf-9788-17491b44e1d6">